common/src/hand_off.rs

```diff
@@ -261,6 +261,18 @@ pub struct FirmwareHandoffTable {
     /// Index of RT SVN value in the Data Vault
     pub rt_svn_dv_hdl: HandOffDataHandle,
 
+    /// LdevId TBS Size.
+    pub ldevid_tbs_size: u16,
+
+    /// FmcAlias TBS Size.
+    pub fmcalias_tbs_size: u16,
+
+    /// PCR log entry count.
+    pub pcr_log_entry_count: u16,
+
+    ///Fuse log entry count.
+    pub fuse_log_entry_count: u16,
```

Comment:
This was the conversation with @ajisaxena about not really needing the log counts since the log array is fixed size and is indexed by the entry id. If this is no more needed, I'll remove the entry counts.

Reply:
IMO it would make more sense to fill the fuse log sequentially and then have a count. Having a sparsely filled out array seems unnecessary. But I could be missing something.

So my recommendation would be:

* If log is filled sequentially, include fuse log count in the FHT
* If not, fuse log count can be omitted

Reply:
Removed the fuse and pcr log counts.